src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -165,6 +165,7 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
 register_cli_argument('webapp config set', 'java_container', help="The java container, e.g., Tomcat, Jetty")
 register_cli_argument('webapp config set', 'java_container_version', help="The version of the java container, e.g., '8.0.23' for Tomcat")
 register_cli_argument('webapp config set', 'app_command_line', options_list=('--startup-file',), help="The startup file for linux hosted web apps, e.g. 'process.json' for Node.js web")
+register_cli_argument('webapp config set', 'arr_affinity', help='the performance of your stateless applications can be improved by turning off the affinity cookie, stateful applications should keep the affinity cookie turned on for increased compatibility', **enum_choice_list(two_states_switch))
```

Comment:
I have no idea what "arr" means. You should either expand the acronym in the help text, or consider aliasing the property to just "affinity" (which is what you call it in the help, after all).

What is "two_states_switch"?  True/False?  Enable/Disable?  If so, I would recommend using `**three_state_switch()` instead of `**enum_choice_list(...)`

Reply:
two state switch is defined as below
two_states_switch = ['true', 'false']
I am not familiar with **three_state_switch(), @tjprescott can you share an example on how to use this? Thank you!